bolt-modules/boltlib/lib/puppet/functions/run_plan.rb

```diff
@@ -11,6 +11,9 @@
   # @param args A hash of arguments to the plan. Can also include additional options.
   # @option args [Boolean] _catch_errors Whether to catch raised errors.
   # @option args [String] _run_as User to run as using privilege escalation.
+  #   This option sets the run-as user for all targets that accept the option
```

Comment:
```suggestion
  #   This option sets the [run-as user](privilege_escalation.md) for all targets that accept the option
```

A link to the privilege escalation doc could be helpful here.

Reply:
Ooo yes good idea